tensorrt_llm/bench/dataset/prepare_dataset.py

```
@@ -35,9 +35,18 @@ class RootArgs(BaseModel):
-            tokenizer = AutoTokenizer.from_pretrained(
-                self.tokenizer, padding_side="left", trust_remote_code=self.trust_remote_code
-            )
+            # PreTrainedTokenizerFast loads directly from tokenizer.json without
+            # invoking AutoConfig, avoiding model-config parsing bugs (e.g.
+            # NemotronHConfig._pattern_to_list KeyError for hybrid models).
+            # Fall back to AutoTokenizer for models that lack a fast tokenizer.
+            try:
+                tokenizer = PreTrainedTokenizerFast.from_pretrained(
+                    self.tokenizer, padding_side="left"
+                )
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)